### ISSUE TITLE
Observer Role

### ISSUE THREAD

**Issue by Emma3DSS** [unitystation/unitystation#2983]
# Observer Role

## Lets players start the round as a ghost instead of a crew member

**Comment by Bod9001** [comment]
This would be useful for Admining Since you need to log on and you don't want to go ssD

**Comment by DMCraftGit** [comment]
Had a look into adding this as a job role. Too many exceptions in character spawning, better off doing it as it's own ghost creation mechanic.

**Comment by DMCraftGit** [comment]
https://github.com/unitystation/unitystation/commit/30ffdd723be5512bd7cba0c022efa4eab8d785f7
I've got it working as an alternative spawn method.

@daalta You were working in the job select UI could I get some help in making it a separate button below the job list? I've just appended it in code to do testing.
![image](https://user-images.githubusercontent.com/61098893/74816149-ad673c80-534e-11ea-9a6e-798c7e864c97.png)
